prime-angular-frontend/src/app/modules/enrolment/pages/regulatory/regulatory.component.ts

```diff
@@ -140,10 +144,11 @@ export class RegulatoryComponent extends BaseEnrolmentProfilePage implements OnI
   protected nextRouteAfterSubmit() {
     const certifications = this.formState.collegeCertifications;
     const careSettings = this.enrolmentFormStateService.careSettingsForm.get('careSettings').value as CareSetting[];
+    const formDeviceProviderIdentifier = this.formState.deviceProviderIdentifier.value;
```

Comment:
Don't need to prefix this with form - `deviceProviderIdentifier`

Reply:
removed **form** prefix